kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/internal/util/Metrics.java

```diff
@@ -280,4 +292,18 @@ public interface ClusterNodeSpecificMetricProviderCreator<T extends Meter> {
 
         MeterProvider<T> create(String virtualCluster, Integer nodeId);
     }
-}
\ No newline at end of file
+
+    /**
+     * Exposes a <a href="https://www.robustperception.io/exposing-the-software-version-to-prometheus/">build info metric</a>  describing Kroxylicious version etc.
+     *
+     * @param versionInfo version info
+     */
+    public static void versionInfoMetric(VersionInfo versionInfo) {
+        Gauge.builder(INFO_METRIC_NAME, ONE_SUPPLIER)
```

Comment:
Should we be adding a similar gauge to the operator metrics? Or does JOSDK already do that for us?